src/index.js

```diff
@@ -164,24 +187,22 @@ class DBLAPI extends EventEmitter {
 
   /**
    * Gets stats from a bot.
-   * @param {string} id The ID of the bot you want to get the stats from.
+   * @param {string} id=client.user.id The ID of the bot you want to get the stats from.
    * @returns {Promise<Object>}
    */
-  async getStats(id) {
-    if (!id && !this.client) throw new Error('getStats requires id as argument');
-    if (!id) id = this.client.user.id;
+  async getStats(id = this._clientID) {
```

Comment:
I already told you in the Discord server that I did not approve of this method to set a default variable.